src/pca_report_library/templates/generate_template.py

```diff
@@ -23,10 +23,8 @@
 from typing import Dict
 
 # Third-Party Libraries
-from docopt import docopt
-
-# cisagov Libraries
 from _version import __version__
+from docopt import docopt
```

Comment:
This needs similar changes as above.

Reply:
This change has been made with commit 358fc3c

Reply:
@ameliav - I don't think this will work:
```
from ._version import __version__
```

Since there is no version file in the templates directory, you probably want:
```
from .._version import __version__
```

Reply:
Ah okay, I see. Change made here 726eb4f9c493d27a85a0bbccd47982334095ea82